packages/fe-mockserver-core/test/unit/data/functionBasedMockData.test.ts

```diff
@@ -33,6 +33,10 @@ describe('Function Based Mock Data', () => {
         );
         let mockData = myEntitySet.getMockData('default').getAllEntries(fakeRequest) as any;
         expect(mockData.length).toBe(3);
+        expect(mockData[0].complexComputedProperty).toBeDefined();
+        expect(mockData[0].complexProperty).toBeDefined();
```

Comment:
These two properties get a value despite being nullable. But that's probably okay as it does not violate the definition 🆗